packages/compiler/test/projection/projector-identity.test.ts

```diff
@@ -285,7 +285,7 @@ describe("compiler: projector: Identity", () => {
           }`,
         ref: "Foo.one",
         expectedTypes: [
-          ["Operation", "one"],
+          ["Operation", "Foo.one"],
```

Comment:
Please help to double check whether this is the expected result. In my understanding, if an operation is in an interface, its name would be namespaceName.interfaceName.operationName. Now it may return namespaceName.operationName.